CONTRIBUTING.md

```diff
@@ -47,7 +126,7 @@ This will clone the main repo to a local repo on your disk and set up the `origi
 Next you will set up the the second side of the triangle to your fork repo.
 
     cd osgi.enroute.site
-    git remote add fork git@github.com:github-user/osgi.enroute.site.git
+    git remote add fork https://github.com/<YourUserName>/osgi.enroute.site.git
```

Comment:
Is there any advantage in doing it over ssh? 

I've read the GitHub [help page](https://help.github.com/articles/which-remote-url-should-i-use/) on this topic and to me it seems ssh comes with quite an overhead compared to https. According to [GitHub Help](https://help.github.com/articles/connecting-to-github-with-ssh/) using ssh requires you to 
* generate the ssh key
* add it to the ssh agent
* add the ssh key to your GitHub account

whereas https works out of the box.

Both methods seem to require a password each time you do a `pull`, `push` or similar to a remote repository, unless your are using a credential helper.

With respect to inexperienced users/contributors (including me ;) ) I would favor a setup as simple as possible.